config/daqconf_full_config.json

```diff
@@ -192,9 +192,33 @@
         "trigger_rate_hz": 1.0,
         "trigger_window_after_ticks": 1000,
         "trigger_window_before_ticks": 1000,
-        "ttcm_s1": 1,
-        "ttcm_s2": 2,
-	"ttcm_prescale": 1,
+        "ttcm_input_map": [
+          {
+            "signal": 0,
+            "tc_type_name": "kTiming",
+            "time_after": 1000,
+            "time_before": 1000
+          },
+          {
+            "signal": 1,
+            "tc_type_name": "kTiming",
+            "time_after": 1000,
+            "time_before": 1000
+          },
+          {
+            "signal": 2,
+            "tc_type_name": "kTiming",
+            "time_after": 1000,
+            "time_before": 1000
+          },
+          {
+            "signal": 3,
+            "tc_type_name": "kTiming",
+            "time_after": 1000,
+            "time_before": 1000
+          }
```

Comment:
this is not an issue but nitpicking, if the purpose of these different signals is to have different readout windows we should reflect that in the default values.